data/schema/game_config.cfg

```diff
@@ -104,6 +104,42 @@
             value="\d+(\.\d+)?-infinity"
         [/element]
     )}
+    # definition of range_tilde_list and s_range_tilde_list, before macros are expanded
+    {LIST_TYPE_COMPLEX range_tilde (
+        [element]
+            value="-?\d+"
+        [/element]
+        [element]
+            value="-?\d+~-?\d+"
+        [/element]
+        [element]
+            value="-?\d+~infinity"
+        [/element]
+        [element]
+            value="-infinity~-?\d+"
+        [/element]
+        [element]
+            value="-infinity~infinity"
```

Comment:
While it fits the rest of the syntax, I feel "allow any number at all" isn't useful as a filter. I'd suggest avoiding any special cases for it, and do whatever is the least code.

Reply:
last case removed and test added